packages/dataviews/CHANGELOG.automattic.md

```diff
@@ -2,6 +2,12 @@
 
 ## Next 
 
+- Expose self-sufficient DataViews subcomponents:
+	- `<DataViews.Search />`
```

Comment:
Wasn't Search already exposed in a previous PR? Also, we could just add the new components to the existing list item below.

Reply:
yes, updating.